### PR TITLE
Fix Cron Script

### PR DESCRIPTION
**Pull request by khalford**
Fixes the cron script not working. Cron runs as root user therefore the variable `$HOME` is not the correct location. Also, the container would be running as root.
Fixes this by switching to the ubuntu user to de-escalate privileges. Then using a shared file location `/etc/chatops` to access docker compose and secrets.

Removed the need to use `chmod +x` in the setup as the file now has file mode `100755`. When cloned the file should already bee executable.

### CODE REVIEW

**Review by DavidFair — DISMISSED** [review]

File: cloud-chatops/INSTALL.md [hunk 1]
```diff
@@ -1,12 +1,72 @@
...
+3. Need to add the Ubuntu user to the docker group for the cron script:
+   ```shell
+   sudo usermod -aG docker ubuntu
+   ```
+4. Move Cron script and start container:
+   ```shell
+   # Copy script to etc/cron.< hourly | daily | monthly >
+   sudo cp /etc/chatops/cloud-docker-images/cloud-chatops/chatopscron /etc/cron.daily/
+
+   # Test the script works and launch the container
```
Comment (Dmitry-Popovichev) [inline cloud-chatops/INSTALL.md:62]:
```suggestion
   # You can test that the script is running correctly by using this command. The --test flag says it won't run the script but it will test if it has been added to the directory correctly
  run-parts --test /etc/cron.<timeframe>
  
  #if you don't receive any output, the script will not run.
  
```

Comment (Dmitry-Popovichev) [reply cloud-chatops/INSTALL.md:62]: Feel free to reword that first comment ` # You can test that...` if you think the suggestion is valid, seems a bit too wordy but I couldn't think of a better way of saying it.

Comment (khalford) [reply cloud-chatops/INSTALL.md]: Changes made.

**Review by Dmitry-Popovichev — APPROVED** [review]
